dashboard/lib/pd/payment/payment_calculator_base.rb

```diff
@@ -152,6 +152,7 @@ def construct_teacher_summaries(workshop_summary, raw_teacher_attendance)
       # Generate a teacher summary for all teachers in raw attendance.
       raw_teacher_attendance.map do |enrollment_id, raw_attendance|
         enrollment = enrollments_by_id[enrollment_id]
+        raise "Unable to find enrollment #{enrollment_id}" unless enrollment
```

Comment:
The next line was blowing up with `undefined method `user_id' for nil:NilClass`. This should not fail again, but just in case we might as well have a more helpful error message.